recipes/cellprofiler-core/meta.yaml

```diff
@@ -22,15 +22,15 @@ requirements:
     - boto3 >=1.12.28
     - centrosome ==1.2.1
     - docutils ==0.15.2
-    - h5py ~=3.6.0
+    - h5py >3.6.0
```

Comment:
The right translation of `~=3.6.0` is `>=3.6.0,<3.7.0`. Same applies for other dependencies